collector/mongos/sharding_status.go

```diff
@@ -221,20 +226,28 @@ func (status *ShardingStats) Describe(ch chan<- *prometheus.Desc) {
 	balancerChunksBalanced.Describe(ch)
 	mongosUpSecs.Describe(ch)
 	mongosPing.Describe(ch)
-	mongosBalancerLockState.Describe(ch)
-	mongosBalancerLockTimestamp.Describe(ch)
+
+	if shared.MongoServerVersionLessThan("3.6", status.Client) {
```

Comment:
why do we use status variable name for stats?

Reply:
Historical thing. :) I've changed it to `s`.